cocotb/share/lib/fli/FliCbHdl.cpp

```diff
@@ -56,7 +56,7 @@ FliTimedCbHdl::FliTimedCbHdl(GpiImplInterface *impl,
 int FliTimedCbHdl::arm_callback()
 {
     #if defined(__LP64__) || defined(_WIN64)
-        mti_ScheduleWakeup64(m_proc_hdl, m_time_ps);
+        mti_ScheduleWakeup64(m_proc_hdl, static_cast<mtiInt32T>(m_time_ps));
     #else
```

Comment:
This seems very odd - the documentation for `mti_ScheduleWakeup64` claims this is a `mtiTime64T`, not a `mtiInt32T`:

![image](https://user-images.githubusercontent.com/425260/73646438-5a646880-4671-11ea-9424-f6849ba9dbe3.png)

Do you have a copy of the header file that says otherwise?

If so, we probably at least want a comment here.

Reply:
It's actually `mtiDelayT` which is `mtiInt32T`.

Reply:
Actually I'll need to check this, I changed it because there was a warning, but it doesn't make much sense for the "64" version of the function.

Reply:
`mtiTime64T` from sources. Will fix later.